dts/arm/nrf52_pca10040.dts

```diff
@@ -24,3 +24,49 @@
 	compatible = "nordic,nrf-uart";
 	current-speed = <115200>;
 };
+
+&flash0 {
```

Comment:
Have you rebased against the latest master? There is already a flash entry in this file in master

Reply:
my mistake